src/Core/Telemetry/IQuickFixesTelemetryManager.cs

```diff
@@ -18,7 +18,7 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
-namespace SonarLint.VisualStudio.IssueVisualization.Editor.QuickActions.QuickFixes
+namespace SonarLint.VisualStudio.Core.Telemetry
```

Comment:
Changed from the base branch: I had to move the manager to Core so it could be used in issueviz and implemented by telemetrymanager in integration.